source/fundamentals/bson.txt

```diff
@@ -210,6 +210,29 @@ Driver will marshal structs using the following rules:
       - Includes an empty ``lastname`` field
       - Stores the ``Address`` field as a nested value
 
+BSON Options
+------------
+
+You can set ``BSONOptions`` to specify how
+the driver marshals and unmarshals BSON on any level, from
+``collection`` to ``Client``.
+
+The following example demonstrates how the ``BSONOptions`` is set using
+the ``Movies`` collection as an example.
```

Comment:
I: In our other docs, we keep collection names lowercase (e.g. [here](https://www.mongodb.com/docs/drivers/node/current/fundamentals/aggregation/#runnable-examples))

S:

```suggestion
The following example demonstrates how the ``BSONOptions`` is set using
the ``movies`` collection as an example.
```